### ISSUE TITLE
pyfloat cant have 0 as digits

### ISSUE THREAD

**Issue by Benjamin-Setzer** [joke2k/faker#401]
fake.pyfloat(0, 2, True)
-> 851420.54

But I expected: 0.XX

the code is as follows:
@classmethod
def pyfloat(cls, left_digits=None, right_digits=None, positive=False):
    left_digits = left_digits or cls.random_int(1, sys.float_info.dig)
    right_digits = right_digits or cls.random_int(0, sys.float_info.dig - left_digits)

compare to random_number:
@classmethod
def random_number(cls, digits=None):
    if digits is None:
        digits = BaseProvider.random_digit()

Basically, I expected pyfloat to behave like random_number and check if digits is None, else use the value I passed.

**Comment by fcurella** [comment]
close by #428

**Comment by Benjamin-Setzer** [comment]
Thank you
